src/pages/Home/index.tsx

```diff
@@ -51,6 +51,14 @@ const Home = () => {
         icon="waving.svg"

```

Comment:
- **Runtime error check**  
  The `AboutContent.title`, `AboutContent.text`, and `AboutContent.section` properties are not checked for null or undefined before being used. This could lead to a runtime error if any of these properties are null or undefined.